Bottom: f5638bcc1b820ecc2fa0d7c8a110e10a0169f266
Top:    832e5d107fc5662b42ab94c8663026dccb16d859
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-08 01:13:45 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 9b2f76e..296eeeb 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
